CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 - `PlayerView.isPictureInPictureRequested` to programatically create a Picture in Picture request
 - `PlayerView.scalingMode` to allow changing the video scaling mode
 - `PlayerViewConfig` with a `UiConfig` to the `PlayerView` to enable configuration of the visual presentation and behaviour
+- `Player.buffer` namespace to control buffer preferences and to query the current buffer state.
```

Comment:
I don't think namespace is the right word here and probably we can just omit it 🙂
```suggestion
- `Player.buffer` to control buffer preferences and to query the current buffer state
```

Reply:
You are right, I also think it makes sense to simplify it.  
Address in de6762a25d004d9d7241d7825c441d297a57e7a6